qp_shotgun/qc_trim/qc_trim.py

```diff
@@ -186,24 +186,11 @@ def _run_commands(qclient, job_id, commands, msg):
     return True, ""
 
 
-def _gzip_file(path):
```

Comment:
Atropos will automatically gzip files that are input as gzipped files, obviating need for this subroutine. 

I do think that this requires gzipping all input fastqs by default further up the pipeline though... We should see if this is already being done.